broker-plugin/plugin/src/main/sh/init-broker.sh

```diff
@@ -90,6 +90,8 @@ function pre_configuration() {
 
     cp $CONFIG_TEMPLATES/bootstrap.xml $BROKER_CONF_DIR/bootstrap.xml
     cp $CONFIG_TEMPLATES/jolokia-access.xml $BROKER_CONF_DIR/jolokia-access.xml
+    cp /opt/broker-plugin/lib/broker-cli.jar $BROKER_CUSTOM/bin/broker-cli.jar
+    cp -r /etc/enmasse-certs $BROKER_CUSTOM/enmasse-certs
```

Comment:
Instead of copying the certs, you could point the script to the JKS trust and keystore generated for the broker and configure the ProtonClientOptions to point to that instead of the PEM.